documentation/src/main/asciidoc/topics/proc_setting_up_target_clusters.adoc

```diff
@@ -1,26 +1,24 @@
+[id='upgrade_target']
 = Setting Up Target Clusters
+Set up a target cluster that runs the {brandname} version to which you want to
+upgrade and add a remote cache store so the target cluster can load data from
+the source cluster.
 
-. Start the target cluster with unique network properties or a different JGroups cluster name to keep it separate from the source cluster.
-. Configure a `RemoteCacheStore` on the target cluster for each cache you want to migrate from the source cluster.
-+
-`RemoteCacheStore` settings::
-+
-* `remote-server` must point to the source cluster via the `outbound-socket-binding` property.
-* `remoteCacheName` must match the cache name on the source cluster.
-* `hotrod-wrapping` must be `true` (enabled).
-* `shared` must be `true` (enabled).
-* `purge` must be `false` (disabled).
-* `passivation` must be `false` (disabled).
-* `protocol-version` matches the Hot Rod protocol version of the source cluster.
-+
-.Example `RemoteCacheStore` Configuration
-[source,xml,options="nowrap",subs=attributes+]
-----
-include::config_examples/remote_cache_store.xml[]
-----
+.Prerequisites
+
+. Download and install the appropriate {brandname} version for the target
+cluster.
+
+.Procedure
+
+. Configure the target cluster with unique network properties or a different
+JGroups cluster name to keep it separate from the source cluster.
+. Add a `RemoteCacheStore` on the target cluster for each cache you want
+to migrate from the source cluster.
 +
-. Configure the target cluster to handle all client requests instead of the source cluster:
-.. Configure all clients to point to the target cluster instead of the source cluster.
-.. Restart each client node.
+Remote cache stores use the Hot Rod protocol to retrieve data from remote
+{brandname} clusters. When you add the remote cache store to the target
+cluster, it can lazily load data from the source cluster to handle client
+requests.
 +
-The target cluster lazily loads data from the source cluster on demand via `RemoteCacheStore`.
+. Switch clients over to the target cluster so it starts handling all requests.
```

Comment:
@gustavonalle you can call `switchToCluster(clusterName)` for Hot Rod Java clients to start sending requests to the target cluster, isn't that so?

the existing docs for rolling upgrade say to update the config and restart clients. but I think that might depend on the client implementation. for Java clients there's no need to restart.

Reply:
I am not familiar with ```switchToCluster(clusterName)```, is this for X-Site probably?

Reply:
yeah, `switchToCluster()` is for xsite but I wonder if it could be used in this scenario as well?